test/OpenTelemetry.Tests.Stress/Skeleton.cs

```diff
@@ -30,6 +30,10 @@ public partial class Program
 
     public static void Stress(int concurrency = 0)
     {
+#if DEBUG
+        Console.WriteLine("***WARNING*** The current build is DEBUG which may affect timing!\n");
```

Comment:
Why do we need `\n` if we already use `WriteLine`?

Reply:
I'd like to add an empty line after the current message, to make it more visible instead of combining it with other benchmark output.

Reply:
Need to use https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-5.0

Reply:
Fixed it in https://github.com/open-telemetry/opentelemetry-dotnet/pull/2437

Reply:
Just curious with one more thought, new line representation is indeed platform dependent, but `Console.WriteLine("\n")` should go to C library as `printf("\n")`. It is unintuitive if `printf("\n")` doesn't work consistently on all platforms, and `printf()` usually does the translation of `\n` to the platform specific new line.

Anyway, `Console.WriteLine("\n")` could choose its own implementation without going to `printf()`.

Reply:
I guess that's why C++ introduced `std::endl`.